### ISSUE TITLE
Simone the Seductress resetting

### ISSUE THREAD

**Issue by only4dank** [elysium-project/bug-tracker#309]
I was doing the Hunter quest 'Stave of the Ancients', and had issues with the demon resetting.

What I would do is trap demon's pet, and then kite Simone while going out of range for when she does cast, then back in to DPS her.

However, she kept on resetting over and over. I even tested being in melee range and getting hit by her lightning bolt. I was not able to keep the fight going more than 20-30 seconds.

Many times I would jump out to out range her cast, then go back in and start shooting her, and she would reset even being 5-15 yard range from me.

I also had issues on the Winterspring demon resetting, however was solved by not kiting her on mountains (figured it had something to do with the elevation... kiting him on the road fixed the resetting there).

Would be nice to get some clarification on what is going on.

BR,
only4dank

**Comment by only4dank** [comment]
Today I tried 2 different methods I saw worked on elysium previously (from early attempt of trapping demon and kiting Simone alone).
1. Trap Simone and kite/kill pet
- Was able to kite the demon but as soon as Simone breaks from trap she resets, and 5-10 seconds after the pet demon then resets as well (regardless of whether I am melee or range attacking it).
2. Kite both at same time
-I would keep them both chasing me without trap. Simone would technically only get hit by multi shot on CD, however she reset first and demon followed almost instantaneously after.

Seems nothing works at this moment. The main issue I see is that as soon as one of them breaks trap it resets regardless of my range from them and then the other one follows shortly after. It is weird though that I cannot kite them both, however, which makes me think their kite range is really low.

Hope this helps speed up the problem solving.
